src/blocks/scribe-plugins/scribe-heading-plugin.js

```diff
@@ -2,26 +2,42 @@
 
 var scribeHeadingPlugin = function(block) {
   return function(scribe) {
-    
-    const headingCommand = new scribe.api.Command('heading');
+
+    const { defaultHeadingLevel, headingLevels } = block.editorOptions;
+    const minHeadingLevel = headingLevels[0];
+    const maxHeadingLevel = headingLevels[headingLevels.length - 1];
```

Comment:
Defining `[2, 3, 6]` will be no different from `[2, 6]`. Perhaps that's fine, but if we are documenting this configuration we should say that the configuration defines a [min, max], rather than a list of allowed heading values.

`[4, 1, 3, 2]` will cause it to alternate between level 4 and nothing. Maybe we should sort the list.

Reply:
We can either use a config for min / max or there might be reasons someone might want [2,4] only. If so I should sort and also increment to the next index rather than the +1 on a level

Reply:
Yeah, looping through the array value is what I meant to get at initially.

Reply:
I'll do this later

Reply:
Normalised the styling with browser defaults at the same time.